src/biome/text/_model.py

```diff
@@ -270,7 +270,10 @@ def predict(self, *args, **kwargs) -> Dict[str, numpy.ndarray]:
 
         inputs = self._model_inputs_from_args(*args, **kwargs)
         instance = self.text_to_instance(**inputs)
-        prediction = self.forward_on_instance(instance)
+        try:
+            prediction = self.forward_on_instance(instance)
+        except Exception as error:
+            raise RuntimeError(f"Failed to make a prediction for '{inputs}'") from error
```

Comment:
Could we use our error exceptions instead. Should fit this error with `WrongValueError` exception ?

Reply:
Done